Src/BootCamp.Chapter/LogUtility/Logger.cs

```diff
@@ -0,0 +1,25 @@
+﻿using BootCamp.Chapter.LogUtility;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class Logger
```

Comment:
I think this class is of unneeded complexity. Simply have 2 loggers: FileLogger, ConsoleLogger. Pass the path to FileLogger so that it knows where to log and you're good to go. Right now even though this adds some flexibility, the amount of confusion and complexity it introduces is not worth it.

Reply:
Changed so that only FileLogger.cs and ConsoleLogger.cs remains and deleted the Logger.cs that use to exist.  The Logger.cs was originally being used to determine which type of logger to use.